ci-operator/config/openshift/openshift-docs/openshift-openshift-docs-no-4.15.yaml

```
@@ -0,0 +1,41 @@
+base_images:
+  openshift-docs:
+    name: openshift-docs-asciidoc
+    namespace: ocp
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: openshift-docs-asciidoc
+    namespace: ocp
+    tag: latest
+  use_build_cache: true
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 200m
+      memory: 400Mi
+tests:
+- as: deploy-preview
+  steps:
+    env:
+      DISTROS: openshift-enterprise openshift-rosa openshift-dedicated microshift
+      PREVIEW_SITE: ocpdocs-pr
+    test:
+    - ref: openshift-docs-build-docs
+    - ref: openshift-docs-preview-comment
+- as: validate-asciidoc
+  steps:
+    env:
+      BUILD: build.py
+      DISTROS: openshift-enterprise
+      VERSION: "4.14"
+    test:
+    - ref: openshift-docs-asciidoctor
+    - ref: openshift-docs-lint-topicmaps
+    - ref: openshift-docs-portal
+zz_generated_metadata:
+  branch: no-4.15
```

Needs to match exactly the branch name

```suggestion
  branch: no-1.5
```